bundles/org.eclipse.passage.loc.billing.core/src/org/eclipse/passage/loc/internal/billing/core/ProductVersionLicense.java

```diff
@@ -0,0 +1,66 @@
+/*******************************************************************************
+ * Copyright (c) 2020 ArSysOp
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0/.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Nikifor Fedorov <zelenyhleb@gmail.com> - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.passage.loc.internal.billing.core;
+
+import java.util.Objects;
+
+import org.eclipse.passage.lic.users.UserLicenseDescriptor;
+
+/**
+ * Local data class designed to sort licenses by product and version.
+ * 
```

Comment:
Why not since 0.8.0?

Reply:
Cause `loc.billing.core` bundle [declares](https://github.com/eclipse/passage/blob/master/bundles/org.eclipse.passage.loc.billing.core/META-INF/MANIFEST.MF) it's version as _0.1.0_. Did I miss something? 

And _patch_-segment does not go to the `@since` tag.